lib/debloater/index.rb

```diff
@@ -56,21 +56,31 @@ def size
 
 
     def debloat!
+      _log "Old index definition:"
+      _log @sql
+
       [
-        "DROP INDEX IF EXISTS idx_debloat_new",
+        'DROP INDEX IF EXISTS idx_debloat_new',
         @sql.
           sub(/CONCURRENTLY/i, '').
-          sub(/CREATE (UNIQUE)? INDEX/i, 'CREATE \1 INDEX CONCURRENTLY').
+          sub(/CREATE\s+(UNIQUE\s+)?INDEX/i, 'CREATE \1INDEX CONCURRENTLY').
           sub(@name, 'idx_debloat_new'),
-        'BEGIN',
-        "ALTER INDEX #{@name} RENAME TO idx_debloat_old",
-        "ALTER INDEX idx_debloat_new RENAME TO #{@name}",
-        'DROP INDEX idx_debloat_old',
-        'COMMIT',
+        %{
+          BEGIN;
+          ALTER INDEX #{@name} RENAME TO idx_debloat_old;
+          ALTER INDEX idx_debloat_new RENAME TO #{@name};
+          DROP INDEX idx_debloat_old;
+          COMMIT;
+        },
```

Comment:
Definitely intended — this means it's sent in one go to the PG server (single roundtrip). Before (array), the round-trip time between "drop" and "commit" would result in many deadlocks.